testdata/p4_16_samples_outputs/psa-table-hit-miss.p4-error

```diff
@@ -0,0 +1,3 @@
+psa-table-hit-miss.p4(62): [--Wwarn=invalid_header] warning: accessing a field of a potentially invalid header hdr.ethernet
```

Comment:
what is this file

Reply:
Well, ironically, when I pushed yesterday, this test failed. I checked the logs and there was a message that this output was expected in the file. This was very strange considering that I didn't change any of the files and the tests were passing before. I will remove this file and try again.